packages/itwinui-css/src/menu/menu.scss

```diff
@@ -100,6 +100,26 @@ $iui-active-outline: thin solid var(--iui-color-border-accent);
   }
 
   &.iui-active {
+    @supports (selector(.iui-active:has(+ .iui-active))) {
+      // Styles active items between the first & last
+      + .iui-active {
+        border-radius: 0;
+      }
+
+      // Styles first active item with an active item following it
+      &:has(+ .iui-active) {
```

Comment:
`iui-active` is not enough by itself, we usually want `iui-menu-item.iui-active` together.

you can write it like this to keep specificity flat and avoid the need to even reference any selector directly:
```scss
    &:where(:has(+ #{&})) {
      border-bottom-left-radius: 0;
      border-bottom-right-radius: 0;
    }

    + :where(#{&}) {
      border-top-left-radius: 0;
      border-top-right-radius: 0;
    }
```

Reply:
Much cleaner, change made.